src/sage/rings/finite_rings/element_base.pyx

```diff
@@ -1,4 +1,4 @@
-# sage.doctest: optional - sage.rings.finite_rings
+# sage.doctest: needs sage.rings.finite_rings
 """
```

Comment:
Aren't all files in `src/sage/rings/finite_rings` installed by the distribution that provides the feature?

Reply:
No, sagemath-categories ships the basic modules of `sage.rings.finite_rings` - https://github.com/mkoeppe/sage/blob/t/32432/modularization_of_sagelib__break_out_a_separate_package_sagemath_polyhedra/pkgs/sagemath-categories/MANIFEST.in.m4#L109-L123

This makes the prime fields and the integer mod rings available.

Reply:
Right. I missed it.